server/db/driver/pg/orders.go

```diff
@@ -1131,6 +1184,41 @@ func findOrder(dbe *sql.DB, oid order.OrderID, fullTable string) (bool, pgOrderS
 	}
 }
 
+func findOrders(ctx context.Context, dbe *sql.DB, aid account.AccountID, oids []order.OrderID, fullTable string) (map[order.OrderID]order.OrderStatus, map[order.OrderID]int64, error) {
```

Comment:
`findOrders` is not quite descriptive of the specialized info this function returns. 

I would also just return a `map[order.OrderID]*db.OrderStatus` instead of separate maps for status and fill.

Reply:
I chose that name to be consistent with the similar function `findOrder` just above that returns the status, type and fill alone as well.